src/components/v5/common/ActionSidebar/partials/Motions/Motions.tsx

```diff
@@ -125,6 +110,14 @@ const Motions: FC<MotionsProps> = ({ transactionId }) => {
                 refetchMotionState={refetchMotionState}
               />
             ) : null,
+          tooltipProps: {
+            tooltipContent: getStakingStepTooltipText(
+              networkMotionState,
+              motionData,
+            ),
+            placement: 'left' as Placement,
```

Comment:
Why was this casting necessary ?

Reply:
The component was expecting a type of `Placement | undefined` and it was recognizing this as not a valid option for `Placement` but just as a `string`, but I think I found a type that we could use instead